Adjust the quilt/setup script to do the equivalent of the GNU file -b option

This patch will be passed upstream

--- quilt/setup.in
+++ quilt/setup.in
@@ -147,7 +147,7 @@ create_md5sums()
 				filetype="xz"
 				;;
 			*)
-				filetype=$(file -b "$file")
+				filetype=$(file "$file" | sed 's/^.*:[\t ]*//')
 				;;
 		esac
 
@@ -402,7 +402,7 @@ case "$1" in
 		"# Source: "*)
 			shift 2
 			source="$@"
-			filetype=$(file -b "$source")
+			filetype=$(file "$source" | sed 's/^.*:[\t ]*//')
 			case "$filetype" in
 			Zip*)
 				echo "unzip ${tar_dir:-.} ${source// /\\ }"
